src/verso/Verso/Code/Highlighted.lean

```diff
@@ -601,6 +601,7 @@ def highlightingStyle : String := "
   text-decoration-style: wavy;
   text-decoration-line: underline;
   text-decoration-thickness: from-font;
+  text-decoration-skip-ink: none;
```

Comment:
This seems unrelated...

Is it to make wavy underlines under e.g. `y` more visible? Good idea, and fine to merge :)

Reply:
Sorry, this must have slipped through when I was splitting the PRs: yes, there was an example in one of the error explanations where the error message appeared only on an identifier `g` and there was no red squiggle. This perhaps makes the rendering a little less pleasing when there are longer spans with a descender but avoids (I think much worse) situations like that one.